plugins/module_utils/nae.py

```diff
@@ -1021,6 +1021,9 @@ def send_pre_change_payload(self):
         self.result['Result'] = "Pre-change analysis %(name)s successfully created." % self.params
 
     def new_object_selector(self):
+        obj = self.query_compliance_object(json.loads(self.params.get('form')).get('name'))
```

Comment:
You might want to do a try/except around the json.loads in case it is not a proper JSON.
You might also want to verify that the return of json.loads is not None to be able to do a get.